fredshema/hoop-mobile-app
# feat: booking implementation

**What does this PR do?**
Save booking info in the appwrite database

**Description of Task to be completed?**
Use appwrite Database sdk to save booking details.

**How should this be manually tested?**
Set the correct environment variables for you appwrite bookings database and start the app. On the home screen click on one parking spot and continue with the booking instructions. After booking check the database for a new document in the database.
 
**What are the relevant pivotal tracker/Trello stories?**
https://trello.com/c/BLw6NJ6K/44

**Questions:**
N/A
